mkdocs.yml

```
@@ -38,10 +38,12 @@ nav:
     - Icons:                              'fundamentals/icons.md'
     - Localization:                       'fundamentals/localization.md'
     - Styles:
+      - Colors:                            'fundamentals/styles/colors.md'
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)